### ISSUE TITLE
When will the v15.0 Business SDK be released? 

### ISSUE THREAD

**Issue by rgtodd** [facebook/facebook-java-business-sdk#400]
When will the v15.0 Business SDK be released?

**Comment by xuantuan58** [comment]
We got the same issue. When the v15.0 released? It's been a month since the API v15.0 released

**Comment by RishabhTayal** [comment]
You can use the v15 like this:

```
<dependency>
  <groupId>com.facebook.business.sdk</groupId>
  <artifactId>facebook-java-business-sdk</artifactId>
  <version>15.0.0</version>
 </dependency>
```

I tested and it's working for me even though they haven't officially released v15 version of the SDK.
